src/components/Search.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="middle-center">
+    <h4>Search:</h4>
+    <b-row>
+      <b-col>
+        <b-form-input
+          v-model="searchQuery"
+          type="text"
+          placeholder="enter movie title"
+        ></b-form-input>
+      </b-col>
+      <b-col>
+        <strong>Status: {{ searchIndicator }}</strong>
+      </b-col>
+    </b-row>
+  </div>
+</template>
+
+<script>
+import _ from "lodash";
+import MovieService from "../services/MovieService";
+export default {
+  props: {
+    searchMovie: Function,
+  },
+  data() {
+    return {
+      searchQuery: "",
+      searchQueryIsDirty: false,
+      isCalculating: false,
+      testMovie: {},
+    };
+  },
+  computed: {
+    searchIndicator: function () {
+      if (this.isCalculating) {
+        return "⟳ Fetching new results";
+      } else if (this.searchQueryIsDirty) {
+        return "... Typing";
+      } else {
+        return "✓ Done";
+      }
```

Comment:
Mene je lepse ovako da se napise :

`if (this.isCalculating) {
        return "⟳ Fetching new results";
    }
  if (this.searchQueryIsDirty) {
        return "... Typing";
      }
        return "✓ Done";
      `